.gitignore

```diff
@@ -6,7 +6,7 @@
 *.xcscmblueprint
 build
 .build
-
+*OAuth*
```

Comment:
I don't think that's a good idea to add and ignore all files that have _OAuth_ in them. Do you have a reason for this?